### PR TITLE
Publish the Ember Data 2.1.0 release blog post

### PR DESCRIPTION
**Pull request by bmac**

### CODE REVIEW

**Comment by mixonic** [comment]
:+1:

File: source/blog/2015-10-05-ember-data-2-1-released.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Ember Data team spent this release cycle focused on improving the
+guides, API documentation and fixing bugs.
+
+See the [Ember Data 2.1.0 CHANGELOG](https://github.com/emberjs/data/blob/v2.1.0/CHANGELOG.md) for a full list of all changes.
+
+## Ember Data 2.2 beta
+
+No new features are currently planned for Ember Data 2.2. During the
+canary cycle leading to this beta, the Ember Data team and community
+have been primarily focused on bug fixes and preformance
```
Comment (sly7-7) [inline source/blog/2015-10-05-ember-data-2-1-released.md]: typo, performance

Comment (bmac) [inline source/blog/2015-10-05-ember-data-2-1-released.md]: Thanks

**Comment by igorT** [comment]
:+1:

**Pull request merged**